org.testeditor.tcl.dsl/src/main/java/org/testeditor/tcl/dsl/jvmmodel/TestRunReporterGenerator.xtend

```diff
@@ -25,8 +29,27 @@ class TestRunReporterGenerator {
 
 	static val logger = LoggerFactory.getLogger(TestRunReporterGenerator)
 
+	private def String getLocationInfo(StepContentElement stepElement) {
+		val element = stepElement.componentElement
+		val nodeModel = NodeModelUtils.findActualNodeFor(element)
+		return '''«element.eResource.URI»:«nodeModel?.startLine»'''
```

Comment:
Can `nodeModel` actually be `null`? If so, that would lead to a somewhat awkward output. Maybe leave off the colon and line number completely when it cannot be determined, or append something like `<unknown line number>`…?